static/downloadables/parse_associations.py

```diff
@@ -424,16 +399,14 @@ def parse_vcf(filteredFilePath, clumpsObjDict, tableObjDict, snpSet, clumpNumDic
                             sample = call.sample
                             genotype = record.genotype(sample)['GT']
                             alleles = formatAndReturnGenotype(genotype, REF, ALT)
+                            complements = takeComplement(alleles, REF, ALT)
 
                             # Grab or create maps that hold sets of unused variants for this sample
                             clumpedVariants = clumped_snps_map[sample] if sample in clumped_snps_map else set()
                             unmatchedAlleleVariants = neutral_snps_map[sample] if sample in neutral_snps_map else set()
-                            atRisk = True if riskAllele in alleles else False
+
+                            atRisk = True if riskAllele in alleles or (complements is not None and riskAllele in complements) or "." in alleles else False #TODO this is an important line that needs to be looked at
```

Comment:
Should this TODO be resolved before pushing? If not, clarify why the line needs to be inspected.

Reply:
I belief it should be good, removed the todo